src/js/components/Select/Select.stories.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Select from "./Select";
+
+const stories = storiesOf("Select", module);
+
+const optionsList = [
+  {
+    label: 'Option 1',
+    value: 'option 1'
+  },
+  {
+    label: 'Option 2',
+    value: 'option 2'
+  },
+  {
+    label: 'Option 3',
+    value: 'option 3'
+  }
+]
```

Comment:
would it be nice to add sub menus in the select?

Reply:
The current React component doesn't support that, do you think that is something we should look at adding in?

Reply:
if it's not supported then don't worry about it.